### PR TITLE
update README to link 1.21 sheet and link issue template

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ If you are not sure, ask someone in the SIG where you initially circulated the i
 
 ## When to Create a New Enhancement Issue
 
-Create an issue here once you:
+Create an [issue](https://git.k8s.io/enhancements/ISSUE_TEMPLATE.md) in this repository once you:
```
Comment (palnabarun) [inline README.md:43]: Can we link to the create issue link instead? https://github.com/kubernetes/enhancements/issues/new

This removes additional clicks on behalf of people wanting to create an issue.

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kikisdeliveryservice) [reply README.md:43]: sure!

Comment (kikisdeliveryservice) [reply README.md]: Done PTAL

Comment (palnabarun) [reply README.md]: Thank you! :)

**Comment by palnabarun** [comment]
Thank you @kikisdeliveryservice 👍🏽 

/lgtm
/approve

**Comment by jeremyrickard** [comment]
Thanks @kikisdeliveryservice 

/approve

**Pull request merged**